test/libsolidity/syntaxTests/parsing/var_assigning_void.sol

```diff
@@ -0,0 +1,16 @@
+contract C {
+    function returnsVoid() pure public { }
+
+    function testVoidAssignmentStmt() pure public {
+        uint i = 42;
+        i = returnsVoid();
+    }
+
+    function testVoidExprInitialization() pure public {
+        uint i = returnsVoid();
+    }
+}
+// ----
+// TypeError: (142-155): Type tuple() is not implicitly convertible to expected type uint256.
+// Warning: (228-250): Different number of components on the left hand side (1) than on the right hand side (0).
```

Comment:
Can you check if we can get rid of the warning? These are basically two messages saying the same thing.